models/src/main/thrift/content/v1.thrift

```diff
@@ -998,6 +998,10 @@ struct ContentFields {
     43: optional string shortSocialShareText
 
     44: optional string socialShareText
+
+    45: optional string bodyText
+
+    46: optional i32 charCount
```

Comment:
We decided not to because the `wordCount` is actually calculated by composer based on the `bodyText`, so the naming is consistent.